src/Sentry/GlobalSessionManager.cs

```diff
@@ -91,6 +104,12 @@ private void DeletePersistedSession()
             return;
         }
 
+        if (_options.DisableFileWrite)
+        {
+            _options.LogInfo("File write has been disabled via the options. Skipping deleting persisted session.");
```

Comment:
"skipping deletion if session files"

when we're reading the debug level log we're not sure how individual messages relate so could be helpful to be explicit